File: Examples/CSharp/AsposePdfFacades/Annotations/AddFreeTextAnnotation.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;
using System.Drawing;

namespace CSharp.AsposePDFFacades.Annotations
{
    public class AddFreeTextAnnotation
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Annotations();
            //open document
            PdfContentEditor contentEditor = new PdfContentEditor();
            contentEditor.BindPdf(dataDir+ "AddFreeTextAnnotation.pdf");
            //crate rectangle
            System.Drawing.Rectangle rect = new System.Drawing.Rectangle(50, 50, 100, 100);
            //create annotation
            contentEditor.CreateFreeText(rect, "Sample content", 1);
            //save updated PDF file
            contentEditor.Save(dataDir+ "AddFreeTextAnnotation_out.pdf");
            
            
        }
    }
}

File: Examples/VisualBasic/AsposePdfFacades/Annotations/AddFreeTextAnnotation.vb
Imports System.IO
Imports System
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades
Imports System.Drawing

Namespace AsposePDFFacades.Annotations
    Public Class AddFreeTextAnnotation
        Public Shared Sub Run()
            ' ExStart:AddFreeTextAnnotation
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_Annotations()
            ' Open document
            Dim contentEditor As New PdfContentEditor()
            contentEditor.BindPdf(dataDir & Convert.ToString("AddFreeTextAnnotation.pdf"))
            ' Create rectangle
            Dim rect As New System.Drawing.Rectangle(50, 50, 100, 100)
            ' Create annotation
            contentEditor.CreateFreeText(rect, "Sample content", 1)
            ' Save updated PDF file
            contentEditor.Save(dataDir & Convert.ToString("AddFreeTextAnnotation_out_.pdf"))
            ' ExEnd:AddFreeTextAnnotation            
        End Sub
    End Class
End Namespace
